src/main/java/yoyo/command/CommandList.java

```diff
@@ -0,0 +1,26 @@
+package yoyo.command;
+
+import yoyo.core.DialogHandler;
+import yoyo.core.Storage;
+import yoyo.exception.YoyoException;
+import yoyo.task.TaskList;
+
+public class CommandList extends Command {
+    public CommandList(String[] inputTokens) {
+        super(inputTokens);
+    }
+
+    /**
+     * Executes "list" command.
+     *
+     * @param tasks   Tasks currently in the Yoyo program.
+     * @param storage Storage instance of the Yoyo program.
+     * @param dialogHandler      Ui instance of Yoyo program.
```

Comment:
The spacing after the @param tasks and @param dialogHandler could be decreased to better fit with the code style standards.
There are a few more such instances across the other classes that you might want to take a look at.